### PR TITLE
Term entry dart to int()

### PR DESCRIPTION
**Pull request by VivianVanilla**
### Description

I simply added a term entry for the `.toInt()` method by adding a new file located .../type-conversion/**toInt**/**toInt.md**. I added an example of how to use the method and followed the style guide and general procedure to the best of my ability. I did this for the Codecademy portfolio assignment in the 'Learn Git and Github' and had more fun than I expected!   

### Issue Solved

Closes Issue #4570 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/dart/concepts/type-conversion/toInt/toInt.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Example
+
+In the following example, the `.toInt()` method is used to turn `myNumber` into an integer value.
+
+```dart
+void main() {
+  var myNumber = 10.4;
+  myNumber = myNumber.toInt();
+  print(myNumber);
```
Comment (mamtawardhani) [inline content/dart/concepts/type-conversion/toInt/toInt.md:35]:
```suggestion
void main() {
  var myNumber = 10.4;
  var result = myNumber.toInt();
  print(result);
}
```

File: content/dart/concepts/type-conversion/toInt/toInt.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+In Dart, the **`.toInt()`** method converts numeric values into integers. This method is the most useful when numeric values need to be treated as integers.
+
+## Syntax
+
+```pseudo
+number.toInt()
+```
+
+- `number`: Chosen numeric value that needs to be converted into an integer value.
```
Comment (mamtawardhani) [inline content/dart/concepts/type-conversion/toInt/toInt.md:25]:
```suggestion
- `number`: It is the chosen numeric value that needs to be converted into an integer value.
```

File: content/dart/concepts/type-conversion/toInt/toInt.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Example
+
+In the following example, the `.toInt()` method is used to turn `myNumber` into an integer value.
```
Comment (mamtawardhani) [inline content/dart/concepts/type-conversion/toInt/toInt.md:29]:
```suggestion
In the following example, the `.toInt()` method is used to convert `myNumber` into an integer value.
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @VivianVanilla! Thank you for contributing to Codecademy Docs!

The entry is well written! I've provided a few suggestions, could you please review and modify those at your earliest convenience?

Thank you!

**Comment by VivianVanilla** [comment]
Thank you so much for the suggestions and help @mamtawardhani ! I believe I've addressed all your concerns :).

**Review by mamtawardhani — APPROVED** [review]
Good for second review

**Comment by VivianVanilla** [comment]
@letsbuildanything Updated the path!

**Review by letsbuildanything — APPROVED** [review]
LGTM! 🚀

**Pull request merged**